docs/api-and-integration/metadata/jira.md

```diff
@@ -0,0 +1,58 @@
+
+![Jira Deployments](jira-deployment.png)
+
+As a note, _Development_ here refers to an environment type, not an environment name. The environment name in Octopus in this example was actually Azure-Dev. See below for more detail on where this becomes important.
+
+An important thing to note is that the link in the image above will link to the deployment in Octopus. In order for Octopus to feed this Url to Jira **you must have the Server Uri configured on the Configuration/Nodes page**.
+
+## Octopus Connect App and the Jira Extension
+
+Something worth pointing out at this point is that the Jira functionality we're demonstrating here is only available in the SaaS version of Jira. There isn't a SaaS requirement on the Octopus side of the integration though, so you can use either a self-hosted or Cloud version of Octopus.
+
+With that in mind, the first step in the integration is to add the Octopus Connect App in the Jira marketplace.
+
+***TODO: add a screenshot of the marketplace tile***
+
+To complete the installation of the app from the marketplace you must configure some security related values. This requires aligning a couple of pieces of information from the page that appears in Jira:
+
+![Jira ConnectApp configuration](jira-app-config.png)
+
+And from the **Configuration / Jira Issue Tracker** page in Octopus:
+
+![Octopus Jira Issue Tracker configuration](octo-jira-config.png)
+
+You'll probably want both of these open in separate browser windows/tabs to complete the configuration. On the Jira side you need to copy the _Octopus Installation Id_ from the Octopus page, and on the Octopus side you'll need to copy your Jira instance's *Base Url* and the *Octopus Secret* that appears on the Connect App configuration page to use as the *Jira Connect App Password*.
+
+## Environment Settings
+
+The next thing to configure relates back to our earlier note on environment types VS environment names. In Octopus you'll need to map the Jira environment types for the environments that you're going to be tracking work-items for:
+
+![Octopus Environment](octo-env.png)
+
+The fixed list of environment types are important for the tracking in Jira and this configuration allows you the flexibility of easily mapping any existing environments to a type.
+
+The environment name does get passed through to Jira, as shown in the earlier example, so the information is still available for you to see. Also note that if you are using multi-tenancy this is fully supported and what you will see in Jira is a per tenant per environment name. As an example, the following diagram illustrates an un-tenanted deployment to Dev and then a tenanted deployment to Dev of the same version (the text is truncated in the shot, the alt text for the link reads `Deploy work-items test release 0.0.5 to Dev for Tenant A`)
```

Comment:
```suggestion
The environment name also gets passed through to Jira, as shown in the earlier example. Also note that multi-tenancy is fully supported and the a deployment will show in Jira per tenant per environment name. 

The following diagram illustrates an un-tenanted deployment to Dev and then a tenanted deployment to Dev of the same version (the text is truncated in the shot, the alt text for the link reads `Deploy work-items test release 0.0.5 to Dev for Tenant A`)
```

It's not clear why we need environment types. What do they give me? Why should I care?

For the screenshot, can we use a shorter project name, so we can see it?